packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -104,26 +142,17 @@ const ProductTile = (props) => {
                 {/* Title */}
                 <Text {...styles.title}>{localizedProductName}</Text>
 
-                {/* Price */}
-                <Text {...styles.price} data-testid="product-tile-price">
-                    {hitType === 'set'
-                        ? intl.formatMessage(
-                              {
-                                  id: 'product_tile.label.starting_at_price',
-                                  defaultMessage: 'Starting at {price}'
-                              },
-                              {
-                                  price: intl.formatNumber(price, {
-                                      style: 'currency',
-                                      currency: currency || activeCurrency
-                                  })
-                              }
-                          )
-                        : intl.formatNumber(price, {
-                              style: 'currency',
-                              currency: currency || activeCurrency
-                          })}
-                </Text>
+                {/*/!* Price *!/*/}
+                <DisplayPrice
+                    basePrice={isProductASet ? listPrice?.maxPrice : listPrice?.price}
+                    isProductASet={isProductASet}
+                    discountPriceProps={
+                        listPrice?.maxPrice > currentPrice || listPrice?.price > currentPrice
+                            ? {as: 'b'}
+                            : {as: 'p'}
```

Comment:
Is the `p` tag the right tag to be using in this scenario? I'm assuming we are only wrapping the price which is never really a paragraph. Would a `span` be a better choice?